### PR TITLE
fix passing city for countries where city is not required

### PR DESCRIPTION
**Pull request by AlexeyKovalkovvv**

### CODE REVIEW

File: splitit-installment-payments-enabler/assets/javascript/splitit-checkout.js [hunk 1]
```diff
@@ -241,6 +241,10 @@
                 var elem = $(this).find('input.input-text').val();
             }
 
+            if ($(this).prop('id') == 'billing_city_field' && !elem) {
+                elem = $('#billing_country option:selected').text();
+            }
```
Comment (sixg) [inline splitit-installment-payments-enabler/assets/javascript/splitit-checkout.js:246]: Has this been tested thoroughly?

**Review by sixg — COMMENTED** [review]
Please test it again on a fresh install.

File: splitit-installment-payments-enabler/assets/javascript/splitit-checkout.js (hunk 1 above)
Comment (AlexeyKovalkovvv) [reply splitit-installment-payments-enabler/assets/javascript/splitit-checkout.js:246]: @sixg 
Yes, I tested it locally and just tested on their staging by putting this js pieces into browser js file, city was passed

Comment (AlexeyKovalkovvv) [reply splitit-installment-payments-enabler/assets/javascript/splitit-checkout.js:246]: There're only two cases to test: when city is filled and not filled for City-country

Comment (sixg) [reply splitit-installment-payments-enabler/assets/javascript/splitit-checkout.js:246]: still successful orders are not placed.

Comment (AlexeyKovalkovvv) [reply splitit-installment-payments-enabler/assets/javascript/splitit-checkout.js:246]: @sixg I tried again, for now it seems to be fine. Please check my comment in the ticket

**Pull request merged**